frontend/src/pages/lesson/InsightsDuring.jsx

```diff
@@ -0,0 +1,256 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import { makeStyles, useTheme } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import { observer } from 'mobx-react-lite';
+import Activity from './Activity';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import Paper from '@material-ui/core/Paper';
+import PercentageDoughnut from '../_shared/PercentageDoughnut';
+import service from '../../service';
+import { math } from 'mathjs';
```

Comment:
Dit geeft een error (Attempted import error: 'math' is not exported from 'mathjs'). Waarschijnlijk werkt dit wel:

`import { median, mean } from 'mathjs';`